client/app/components/index.jsx

```diff
@@ -2,15 +2,55 @@ import React from 'react';
 import {render} from 'react-dom';
 import Query from './query.jsx';
 import Gmap from './gmap.jsx';
+import $ from 'jquery';
```

Comment:
We should consider doing some more research on http libraries for react - check out this link https://daveceddia.com/ajax-requests-in-react/